webhook/validator/pipeline.go

```diff
@@ -58,6 +79,28 @@ func (v *pipelineValidator) ValidateUpdate(ctx context.Context) *admissionv1.Adm
 	if v.newPipeline.Spec.InterStepBufferServiceName != v.oldPipeline.Spec.InterStepBufferServiceName {
 		return DeniedResponse("Cannot update pipeline with different interStepBufferServiceName")
 	}
+	// Check that the ISB service exists
```

Comment:
Can skip the check here. It's not allowed to update isb svc name, so we can assume it is existing.